CHANGELOG.rst

```diff
@@ -2,6 +2,54 @@
 CHANGELOG
 =========
 
+2.8.0
+=====
+
+**ENHANCEMENTS**
+
+* Enable support for ARM instances on Ubuntu 18.04 and Amazon Linux 2.
+* Add support for the automatic backup features of FSx file systems.
+* Renewed user experience and robustness of cluster update functionality.
+* Support DCV and EFS in China regions.
+* Use DescribeInstanceTypes API to validate whether an instance type is EFA-enabled so that new EFA instances can
+  be used without requiring an update to the ParallelCluster configuration files.
+* Enable Slurm to directly launch tasks and initialize communications through PMIx v3.1.5 on all supported
+  operating systems except for CentOS 6.
+* Print a warning when using NICE DCV on micro or nano instances.
+
+**CHANGES**
+
+* Remove the client requirement to have berks (or ChefDK) to build a custom AMI.
```

Comment:
berks -> Berkshelf  
please remove the "(or ChefDK)" that we're not mentioning it anymore as dependency in our doc here  https://docs.aws.amazon.com/parallelcluster/latest/ug/pcluster.createami.html